### PR TITLE
Service/dlp: Update function tables according 3dbrew

### PR DESCRIPTION
**Pull request by mailwl**
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/2864)

### CODE REVIEW

File: src/core/hle/service/dlp/dlp_clnt.cpp [hunk 1]
```diff
@@ -8,7 +8,26 @@ namespace Service {
...
+    {0x000B0100, nullptr, "PrepareForSystemDownload"},
+    {0x000C0000, nullptr, "StartSystemDownload"},
+    {0x000D0100, nullptr, "StartTitleDownload"},
+    {0x000E0000, nullptr, "GetMyStatus"},
+    {0x000F0040, nullptr, "GetConnectingNodes"},
+    {0x00100040, nullptr, "GetNodeInfo"},
+    {0x00110000, nullptr, "GetWirelessRebootPassphrase"},
+    {0x00120000, nullptr, "StopSession"},
+    {0x00130100, nullptr, "GetCupVersion"},
+    {0x00140100, nullptr, "GetDupAvailability"},
```
Comment (wwylele) [inline src/core/hle/service/dlp/dlp_clnt.cpp]: `0x00140010` unless you can prove 3dbrew is wrong

File: src/core/hle/service/dlp/dlp_srvr.cpp [hunk 2]
```diff
@@ -24,14 +24,19 @@ const Interface::FunctionInfo FunctionTable[] = {
     {0x00010183, nullptr, "Initialize"},
     {0x00020000, nullptr, "Finalize"},
     {0x00030000, nullptr, "GetServerState"},
+    {0x00040000, nullptr, "GetEventDescription"},
     {0x00050080, nullptr, "StartAccepting"},
+    {0x00060080, nullptr, "EndAccepting"},
```
Comment (wwylele) [inline src/core/hle/service/dlp/dlp_srvr.cpp]: `0x00060000` unless you can prove 3dbrew is wrong

Comment (mailwl) [inline src/core/hle/service/dlp/dlp_srvr.cpp]: copy/paste error...

File: src/core/hle/service/dlp/dlp_clnt.cpp (hunk 1 above)
Comment (mailwl) [inline src/core/hle/service/dlp/dlp_clnt.cpp]: looks like 3dbrew error, GetDupAvailability get same amount of parameters, as GetCupVersion, GetTitleInfo and PrepareForSystemDownload

dlp handler don't check headers at all

Comment (mailwl) [inline src/core/hle/service/dlp/dlp_clnt.cpp]: here example:
```
    case 0xB:
      result_1 = PrepareForSystemDownload((int)v1, &v56, v2 + 4, v3, *(_DWORD *)(v2 + 12), *(_DWORD *)(v2 + 16));
      func = 0xB;
      ResultCode = result_1;
      goto LABEL_32;
    case 0x14:
      result_2 = GetDupAvailability((int)v1, &v56, v2 + 4, v3, *(_DWORD *)(v2 + 12), *(_DWORD *)(v2 + 16));
      func = 0x14;
      ResultCode = result_2;
LABEL_32:
```

Comment (wwylele) [inline src/core/hle/service/dlp/dlp_clnt.cpp]: Fine then. You can take this chance to fix 3dbrew

**Pull request merged**